clientd3d/color.c

```diff
@@ -403,19 +408,38 @@ HBRUSH DialogCtlColor(HWND hwnd, HDC hdc, HWND hwndChild, int type)
 * GetItemListColor:  Get given color id # for given owner-drawn list box.
 *    (Inventory has different colors than popup dialog lists)
 *    Doesn't return color itself so that caller can use id to call GetBrush.
+*    Also colors special items.  When the client receives a OF_ITEM_SPECIAL flag
+*    for an object, the client will color the object's text in lists based on the
+*    value of special_type and the corresponding COLOR_ITEM_SPECIAL in color.h
 */
-WORD GetItemListColor(HWND hwnd, int type)
+WORD GetItemListColor(HWND hwnd, int type, int flags, enum special_type special_type_value)
```

Comment:
You don't "enum" here (I didn't even know you could put it here actually).

Reply:
changed `enum` to `typedef enum` in `proto.h` as suggested and adjusted related function parameters